air/src/execution_step/boxed_value/stream.rs

```diff
@@ -149,10 +149,6 @@ impl Stream {
     }
 
     pub(crate) fn is_empty(&self) -> bool {
-        if self.values.is_empty() {
-            return false;
-        }
-
         self.values.iter().all(|v| v.is_empty())
```

Comment:
The original behavior is incorrect, and this code actually isn't used, because `stream.values` is never empty. Looks like this code is needed before when the behavior was a bit different.

Reply:
I don't think so, they are non empty only thanks to the current code, but it's not an architectural design. One could write a method that could use it oppositely.

I mentioned emptiness only because this `if` was incorrect, and it doesn't break because it wasn't used.

Reply:
I see.